modules/base/src/backend/index.ts

```diff
@@ -0,0 +1,29 @@
+import * as sdk from 'botpress/sdk'
+
+export type Extension = {}
+
+export type SDK = typeof sdk & Extension
+
+const onServerStarted = async (bp: SDK) => {}
```

Comment:
let's actually start using `typeof sdk` directly instead of exporting SDK since there's no more notion of mutating the global BP

Reply:
good point, copied the index content from another module and forgot to change it